### PR TITLE
Fix position of search button on tablets.

### PR DESCRIPTION
**Pull request by mpelzsherman**
This PR fixes two issues:

1. [mobile layout issues due to dependabot upgrade of `ismobilejs`](https://github.com/department-of-veterans-affairs/va.gov-team/issues/10749)
1. [position of search button on tablets](https://github.com/department-of-veterans-affairs/va.gov-team/issues/5999)

This PR actually removes the `ismobilejs` dependency.

## Description

The Search button is cut off on iPad, and on mobile, the form extends off the screen and can't be used.

## Testing done

View the Facility Search page in Desktop, iPad, and Mobile clients, refreshing the page each time.

#### Desktop

Find_VA_Locations___Veterans_Affairs

#### iPad

Find_VA_Locations___Veterans_Affairs

#### Mobile

Find_VA_Locations___Veterans_Affairs

## Acceptance criteria
- [x] Search button and overall search form looks good in all 3 devices.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by brangi** [comment]
@mpelzsherman  can you provide some screenshots from your local? I would like to QA this on the review instance, however e2e tcs are failing

File: README.md [hunk 1]
```diff
@@ -115,7 +115,13 @@ yarn test:unit src/applications/path/to/tests/**/*.unit.spec.js*
...
 
 ### Browser tests
-To **run all browser tests**, you first need two things:
+To **run all browser tests**, you first need three things:
+1. Install the Java JDK on MacOS:
+    ```
+    brew update
+    brew tap adoptopenjdk/openjdk
+    brew cask install adoptopenjdk8
+    ```
```
Comment (cvalarida) [inline README.md:124]: I don't think this is necessary, but I'm on Linux. cc @rianfowler

Comment (mpelzsherman) [reply README.md:124]: Java is NOT installed on MacOS by default. I think it used to be once upon a time. (?)

Comment (mpelzsherman) [reply README.md:124]: https://www.forbes.com/sites/adriankingsleyhughes/2012/10/19/apple-makes-os-x-safer-by-removing-java/

**Review by brangi — APPROVED** [review]
LGTM

**Comment by mpelzsherman** [comment]
Breakpoints for desktop are still not quite working ideally, though this is an existing issue on production:

https://recordit.co/KqFG7fwo5p

We should not be using Javascript to render different layouts. All of the responsive design can and should be done with CSS, but this would require extensive refactoring of the existing (legacy) code and is outside the scope of this PR.

**Comment by mpelzsherman** [comment]
Need to fix pagination layout, which is somehow broken in this PR:

Find_VA_Locations___Veterans_Affairs

**Comment by mpelzsherman** [comment]
@mmiddaugh confirmed that the desktop responsive issue does not need to be address in this PR.

File: README.md (hunk 1 above)
Comment (cvalarida) [reply README.md:124]: Talked with some other folks on the front end team and it sounds like I'm wrong. :+1: 

I like your idea (in Slack) about adding the "if you don't already have it" bit, but it's probably fine as-is.

**Review by cvalarida — APPROVED** [review]

**Pull request merged**